src/aiida_vasp/calcs/neb.py

```diff
@@ -51,23 +51,23 @@ def define(cls, spec):
         # options is passed automatically.
         spec.input(
             'parameters',
-            valid_type=get_data_class('core.dict'),
+            valid_type=orm.Dict,
```

Comment:
Have all `get_data_class` been changed to `orm.AiiDAType`?

Reply:
No yet, I should change them.

Reply:
Good call, there are still quite a few left. I will change them.

Reply:
Thank you. It can be done by another PR.